### PR TITLE
Dev server host/port

### PR DESCRIPTION
**Pull request by magnusmanske**
Partially solving #463

### CODE REVIEW

File: webapp/dist/index.html [hunk 1]
```diff
@@ -24,9 +24,9 @@
   <script type="text/javascript">
     //These will be injected at the server
     initialConfig = {
-      serverURL: "http://localhost:8000/api",
+      serverURL: "http://52.48.251.182:8000/api",
```
Comment (benjeffery) [inline webapp/dist/index.html]: Can you back out these changes? Promise #463 and #462 coming soon!

File: webapp/dist/index.html [hunk 2]
```diff
@@ -38,5 +38,5 @@
   <div id="widthWarning">Panoptes requires a screen width of at least 750 pixels.</div>
   <script>document.getElementById('body').removeChild(document.getElementById('JSwarning'))</script>
   <div id="main" class="main"></div>
-<script type="text/javascript" src="panoptes.js?60d5fe63b3e20fc2b018"></script><script type="text/javascript" src="babel.js?60d5fe63b3e20fc2b018"></script></body>
+<script type="text/javascript" src="panoptes.js?f380728fee35f24f956c"></script><script type="text/javascript" src="babel.js?f380728fee35f24f956c"></script></body>
```
Comment (benjeffery) [inline webapp/dist/index.html]: Same here

File: webapp/gulp/config.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ module.exports = {
   server: {
     settings: {
       root: dest,
-      host: 'localhost',
+      host: '0.0.0.0',
```
Comment (benjeffery) [inline webapp/gulp/config.js]: I'd prefer localhost as the default here so people don't accidentally expose stuff

File: webapp/src/index.html [hunk 4]
```diff
@@ -24,9 +24,9 @@
   <script type="text/javascript">
     //These will be injected at the server
     initialConfig = {
-      serverURL: "http://localhost:8000/api",
+      serverURL: "http://52.48.251.182:8000/api",
```
Comment (benjeffery) [inline webapp/src/index.html]: Same here!

**Comment by benjeffery** [comment]
Nice! Just a couple of things specific to your setup I think we shouldn't merge.

**Comment by benjeffery** [comment]
You want me to fix those or you happy to?

**Comment by magnusmanske** [comment]
Should be fixed now.

**Comment by benjeffery** [comment]
This solves #462 now.

**Pull request merged**